components/prombench/manifests/benchmark/3_prometheus-test.yaml

```diff
@@ -733,26 +733,39 @@ spec:
                 - prometheus
       securityContext:
         runAsUser: 0
+      initContainers:
+      - name: prometheus-builder
+        image: docker.io/geekodour/prometheus-builder:2.2.0
+        args: ["{{ .PR_NUMBER }}"]
+        volumeMounts:
+        - name: usrbin-volume
+          mountPath: /usrbin
       containers:
       - name: prometheus
-        image: docker.io/prombench/prometheus-builder:2.0.0
+        image: docker.io/prom/busybox:latest
         imagePullPolicy: Always
-        # The prometheus-builder takes a while to build and run
-        # so make sure we start it before the release deployment.
-        # Mark it ready only when prometheus is started.
-        # This way we have the least time difference in the scraped metrics.
```

Comment:
This comment is good to explain why we need the readiness probe. Can change the wording if needed, but we need some explanation on why do we need a `readinessProbe`